.github/workflows/test.yml

```diff
@@ -15,6 +15,12 @@ jobs:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
     if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/master'"
 
+  test-supervisor:
```

Comment:
should these tests always run or only when the supervisor has been modified?

Reply:
Only when it is modified, should I add this         

if: "env.GIT_DIFF != ''"

condition?

Reply:
you would need to have something like:

```
      - uses: technote-space/get-diff-action@v1
        id: git_diff
        with:
		  PREFIX_FILTER: |
			supervisor
          SUFFIX_FILTER: |
            .go
            .mod
            .sum
```

then the if condition, im not sure if this is correct so you will have to test it